src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Shape implements Figure {
```

Comment:
and add an interface for getArea method
```suggestion
public class Figure implements Drawable {
```